pkg/values/list.go

```diff
@@ -0,0 +1,49 @@
+package values
+
+import (
+	"github.com/smartcontractkit/chainlink-common/pkg/values/pb"
+)
+
+type List struct {
+	Value []Value
```

Comment:
Must all elements be the same type of `Value`?

Reply:
No, there's no requirement for that